cgmanifest.json

```
@@ -0,0 +1,24 @@
+{
+    "Registrations": [
+        {
+            "Component": {
+                "Type": "git",
+                "Other": {
+                    "RepositoryUrl": "https://github.com/bilke/cmake-modules",
+                    "CommitHash": "66fa2579d36f30cdab93d1dbd4c3d682a25ae5f3"
+                }
+            },
+            "DevelopmentDependency": false
+        }, 
+        {
+            "Component": {
+                "Type": "git",
+                "Other": {
+                    "RepositoryUrl": "https://github.com/xbmc/libssh",
+                    "CommitHash": "3cb0a1bf023cce7209290f998b8dc3029ed3df77"
```

nit: Alternatively, for libssh, use, what we have in the comment:
https://github.com/vhvb1989/azure-sdk-for-c/blob/4805a1c1ab3e36234f3e7d803fb3c07aecdb5b38/cmake-modules/AddTestCMocka.cmake#L8

```suggestion
                    "CommitHash": "667fb5f9a9c96f210583dbfb11755c43250c5e55"
```